### PR TITLE
create external cluster endpoint

### PR DESCRIPTION
**Pull request by zreigz**
**What this PR does / why we need it**: create external cluster endpoint

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

```release-note
the new endpoint to create an external cluster: POST /api/v2/projects/{project_id}/kubernetes/clusters
```

### CODE REVIEW

File: pkg/handler/v2/external_cluster/external_cluster.go [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+			return nil, errors.NewBadRequest(err.Error())
+		}
+
+		cfg, err := clientcmd.Load(config)
+		if err != nil {
+			return nil, common.KubernetesErrorToHTTPError(err)
+		}
+
+		if _, err := clusterProvider.GenerateClient(cfg); err != nil {
+			return nil, errors.NewBadRequest(fmt.Sprintf("can not connect to the kubernetes cluster: %v", err))
```
Comment (floreks) [inline pkg/handler/v2/external_cluster/external_cluster.go]: s/can not/cannot

Comment (zreigz) [reply pkg/handler/v2/external_cluster/external_cluster.go]: done

**Comment by zreigz** [comment]
@floreks PTAL

**Pull request merged**